data/brands/amenity/fuel.json

```diff
@@ -1079,6 +1089,20 @@
         "name": "Crodux"
       }
     },
+    {
+      "displayName": "Crown",
+      "locationSet": {
+        "include": ["us"]
+      },
+      "matchNames": ["crown"],
```

Comment:
You won't need a "_matchName_" here as it's the same name as in the "_displayName_" and the "_build_script.js_" can handle names that are the same but different case 👍

Reply:
Just learned, I will fix that (mobile now).

Reply:
It actually won't match the `displayName` but it will match the `name` tag, so yes "crown" is redundant in this case.

Reply:
Ops, my mistake.